compiler-core/src/ast.rs

```diff
@@ -420,6 +421,20 @@ impl<T> Import<T> {
     pub(crate) fn alias_location(&self) -> Option<SrcSpan> {
         self.as_name.as_ref().map(|(_, location)| *location)
     }
+
+    pub(crate) fn cmp(&self, other: &Self) -> Ordering {
```

Comment:
This name looks like the Ord trait, which is a bit misleading.

What's the special case of `gleam/` for? I couldn't see a test that demonstrates what it does

Reply:
It's so that it can put `gleam/` imports on top of each group, the test is this one: <https://github.com/gleam-lang/gleam/pull/2601/files#diff-25116aa6d184c5ec1b2f472955472a17d316f0d04d79fa79f0a1de318fd38faaR41-R49>